coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/Calcite2TrinoUDFConverter.java

```diff
@@ -169,6 +189,57 @@ public RexNode visitCall(RexCall call) {
       return super.visitCall(modifiedCall);
     }
 
+    private Optional<RexNode> visitFromUtcTimestampCall(RexCall call) {
+      RelDataType inputType = call.getOperands().get(0).getType();
+      RelDataType targetType = call.getType();
+
+      List<RexNode> convertedOperands = visitList(call.getOperands(), (boolean[]) null);
+      RexNode sourceValue = convertedOperands.get(0);
+      RexNode timezone = convertedOperands.get(1);
```

Comment:
Wrap timezone into a canonicalize function call (`$canonicalize_hive_timezone_id`), so that we can in the future extend support to other Hive timezone identifiers which are not supported by Trino today.

This function would be identity for now.